plugins/org.jboss.tools.jmx.remoting1_1_3/.classpath

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
isnt' these not already in server as jars anyway ? seems we should make sure this gets done in a way so they can be reused instead of we keep adding jboss AS jars runtime jars.